index.bs

```diff
@@ -644,49 +644,44 @@ The 'Sec-CH-UA-Platform-Version' Header Field {#sec-ch-ua-platform-version}
 ----------------------------------
 
 The <dfn http-header>`Sec-CH-UA-Platform-Version`</dfn> request header field gives a server
-information about the platform version on which a given [=user agent=] is executing. It is a
-[=Structured Header=] whose value MUST be a [=structured header/string=]
-[[!RFC8941]].
+information about the [=user agent/platform version=] on which a given [=user agent=] is executing.
+It is a [=Structured Header=] whose value MUST be a [=structured header/string=][[!RFC8941]]. Its
+value is the result of [=getting the platform version=] with [=user agent/platform brand=].
 
-[=User agents=] SHOULD return a version with a platform-specific format that allows differentiating
-significant platform versions by running these steps:
-    1. If the platform is Linux:
+To <dfn>get the platform version</dfn>, given |platform|, run the following steps:
```

Comment:
Should |platform| be a string? We don't really define the type of platform brand, but a string seems appropriate.

Reply:
Yeah, probably. Not sure any other type makes sense anyways.